app/Http/Controllers/ThirdParty/CallPowerController.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace Chompy\Http\Controllers\ThirdParty;
+
+use Illuminate\Http\Request;
+use Chompy\Services\Rogue;
+use Chompy\Http\Controllers\Controller;
+
+class CallPowerController extends Controller
+{
+
+    /**
+     * Create a controller instance.
+     *
+     * @return void
+     */
+    public function __construct(Rogue $rogue)
+    {
+      // @TODO: add role middleware?
+    	$this->rogue = $rogue;
+
+    }
+
+    /**
+     *
+     * @param Request $request
+     */
+    public function store(Request $request)
+    {
+    	$request->validate([
+    		// QUESTION - We shouldn't have to validate phone number because this will come from Twilio and phone numbers are validated in NS?
+            'mobile' => 'required',
```

Comment:
We shouldn't have to validate phone number because this will come from Twilio and phone numbers are validated in NS, right?

Reply:
Yeah, I think that's a good assumption!